src/EA.Weee.RequestHandlers.Tests.Unit/Organisations/OrganisationByIdHandlerTests.cs

```diff
@@ -260,25 +265,33 @@ public async Task HandleAsync_GivenOrganisationHasNoDirectRegistrant_Organisatio
         [Fact]
         public async Task HandleAsync_GivenOrganisationHasDirectRegistrant_OrganisationShouldHaveDirectRegistrant()
         {
+            // Arrange
             var organisation = GetOrganisationWithId(organisationId);
-
             var expectedReturnValue = new OrganisationData();
             A.CallTo(() => map.Map(A<Organisation>._)).Returns(expectedReturnValue);
 
             var directRegistrant = A.Fake<DirectRegistrant>();
-            A.CallTo(() => directRegistrant.Id).Returns(Guid.NewGuid());
+            var directRegistrantId = Guid.NewGuid();
+
+            A.CallTo(() => directRegistrant.Id).Returns(directRegistrantId);
             A.CallTo(() => directRegistrant.OrganisationId).Returns(organisation.Id);
 
-            var directRegistrants = new List<DirectRegistrant> { directRegistrant };
+            A.CallTo(() => directRegistrant.DirectProducerSubmissions).Returns(new List<DirectProducerSubmission>
+            {
+                new DirectProducerSubmission { ComplianceYear = DateTime.UtcNow.Year }
+            });
 
+            var directRegistrants = new List<DirectRegistrant> { directRegistrant };
             A.CallTo(() => context.DirectRegistrants).Returns(dbHelper.GetAsyncEnabledDbSet(directRegistrants));
 
             var message = new GetOrganisationInfo(organisationId);
 
+            // Act
             var result = await handler.HandleAsync(message);
 
+            // Assert
             result.HasDirectRegistrant.Should().BeTrue();
-            result.DirectRegistrantId.Should().Be(directRegistrant.Id);
+            result.DirectRegistrants.Should().ContainSingle(dr => dr.DirectRegistrantId == directRegistrantId);
```

Comment:
@will-hope98  need an assert to check the submission started

Reply:
Done